libs/shared/ui/src/lib/styles/components/button.scss

```diff
@@ -57,6 +57,14 @@
   }
 }
 
+.btn--confirm {
+  color: white;
```

Comment:
Can you add directly an variable? because for our futur subject the "dark mode" it should be necessary